routes/campaigners.rb

```diff
@@ -20,6 +29,54 @@ class Campaigners < Cuba
       res.redirect "/"
     end
 
+    on "petitions/new" do
+      render("petition/new",
+             title: "Actionite | Create New Petition",
```

Comment:
To not have to repeat "Actionite | " in the `title:` parameter, we can write the <title> like this in the master templates:
`<title>Actionite | {{ title }}</title>`